test/testvm.py

```diff
@@ -83,22 +83,10 @@ def __init__(self, address=None, flavor=None, system=None, arch=None, verbose=Fa
         self.verbose = verbose
         self.flavor = flavor or DEFAULT_FLAVOR
 
-        conf_file = "guest/%s.conf" % self.flavor
-        if os.path.exists(conf_file):
-            with open(conf_file, "r") as f:
-                self.conf = json.load(f)
-        else:
-            self.conf = { }
-
-        self.os = system or self.getconf('os') or testinfra.OS
+        self.os = system or testinfra.OS
         self.arch = arch or testinfra.ARCH
 
-        self.tag = "0"
-        tags = self.getconf('tags')
-        if tags and self.os in tags:
-            self.tag = tags[self.os]
-
-        self.image = "%s-%s-%s-%s" % (self.flavor, self.os, self.arch, self.tag)
+        self.image = "%s-%s-%s" % (self.flavor, self.os, self.arch)
         self.test_dir = os.path.abspath(os.path.dirname(__file__))
         self.test_data = os.environ.get("TEST_DATA") or self.test_dir
```

Comment:
with the new files in `test/images`, we need to get the filenames from that directory - if `TEST_DATA` is set to something else in the environment, the current decision logic won't be able to find the original image

We get the link / image to use from the git repo, which shouldn't be written into the `TEST_DATA` directory.